std/assembly/util/string.ts

```diff
@@ -1,6 +1,412 @@
 import { itoa, dtoa, itoa_stream, dtoa_stream, MAX_DOUBLE_LENGTH } from "./number";
 import { ipow32 } from "../math";
 
+// Lookup table for speedup check is char alpha or not
```

Comment:
Can you extend this comment a bit on how this, the following lookup tables and `stagedBinaryLookup` below work?

Reply:
Yeah. Will do